data/production-email.template

```diff
@@ -17,7 +17,8 @@ this certificate with a newer one that covers more or fewer names than
 the list above, you may be able to ignore this message.
 
 If you want to stop receiving all email from this address, click
```

Comment:
Maybe change to `If you want to stop receiving all email (not only regarding the domains above) from this address, click`